Auto with Golden Tests – an Auto PR in which the tests have been replaced with those from the Golden PR

bdhadage23 opened the pull request:
- Replaced Golden Test file
- Agent not did changes in data.sql so I did to execute the test in agent genrated code.

bdhadage23 committed:
- Auto Roo Code Agent

bdhadage23 committed:
- Added Golden Test in Auto

Gowrishbyndoor reviewed (commented):
After addressing the comment from the golden branch, please add those tests and re-run the tests

bdhadage23 commented:
> After addressing the comment from the golden branch, please add those tests and re-run the tests

Added updated tests from golden branch and tested. Its running successfully on agent generated code.
